packages/client/src/__tests__/dataTriggers.test.ts

```diff
@@ -286,6 +286,60 @@ describe("Cloud functions -> Data triggers ->", () => {
         expect(existingSecrets).toEqual(["anotherSecret"]);
       }
     );
+    testWithEmulator(
+      "should update 'smtpConfigured' in organization data document when smtp config is set in secrets document",
+      async () => {
+        const { organization } = await setUpOrganization(true, false);
+        const organizationPath = `${Collection.Organizations}/${organization}`;
+        const secretsPath = `${Collection.Secrets}/${organization}`;
+        // add new secret to trigger registering
+        const orgSecretsRef = adminDb.doc(secretsPath);
+        await orgSecretsRef.set({
+          smtpHost: "localhost",
+          smtpPort: 4000,
+          smtpUser: "user",
+          smtpPass: "password",
+        });
+        // check proper updates triggerd by write to secrets
+        const orgData = (await waitForCondition({
+          documentPath: organizationPath,
+          condition: (data) => Boolean(data?.existingSecrets?.length),
+        })) as OrganizationData;
+        expect(orgData.existingSecrets).toEqual(
+          expect.arrayContaining([
+            "smtpHost",
+            "smtpPort",
+            "smtpUser",
+            "smtpPass",
+          ])
+        );
+        expect(orgData.smtpConfigured).toEqual(true);
+      }
+    );
+    testWithEmulator(
+      "should not update 'smtpConfigured' in organization data document if not all smtp config is set in secrets document",
+      async () => {
+        const { organization } = await setUpOrganization(true, false);
```

Comment:
These two tests could be just one:
- check it's false with no smtp config
- add one of smtp properties
- check it's still false
- add all of smtp properties
- check it's true
- remove one property
- check it's false again (this last one wouldn't really happen in production, but it makes sense to test it anyway)

the test name could just be `"updates 'smtpConfigured' with respect to smtp config being present in organization data"`